### PR TITLE
33 social impact beyond business section

### PR DESCRIPTION
**Pull request by RajabKarume**
# Description

This pull request is part of the work to make it easier for people to view a section that described the organization beyond business

To achieve this, we needed to:

```
- Create a beyond business component
- Add the correct inputs for the title, paragraph, and image
```

Fixes [#33](https://github.com/italanta/elewa-group/issues/33)

## Type of change

Please delete options that are not relevant.

- [✅] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

Screenshot from 2023-02-23 09-14-06

Screenshot from 2023-02-23 09-14-26

# Checklist:
- [✅] My code follows the style guidelines of this project
- [✅] I have performed a self-review of my code
- [✅] I have commented my code, particularly in hard-to-understand areas
- [✅] I have made corresponding changes to the documentation
- [✅] My changes generate no new warnings
- [✅] I have added tests that prove my fix is effective or that my feature works
- [✅] New and existing unit tests pass locally with my changes
- [✅] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 1]
```diff
@@ -3,9 +3,11 @@ import { CommonModule } from '@angular/common';
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
 
 import { SocialImpactRoutingModule } from './social-impact.routing';
+import { BeyondBusinessComponent } from './pages/components/beyond-business/beyond-business.component';
+import { BannersModule } from '@elewa-group/features/components/banners';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:7]: Order of imports

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

File: libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html [hunk 2]
```diff
@@ -1 +1 @@
-<p>social-impact-page works!</p>
+<elewa-group-beyond-business></elewa-group-beyond-business>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html:1]: You're missing some updates from your peers on this file, please sync

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
This section has a white background please pass the background color as part of the input properties for this component

File: libs/pages/elewa/social-impact/src/lib/pages/components/beyond-business/beyond-business.component.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<elewa-group-elewa-group-image-and-text-banner
+[imagePlacement]="'left'"
+[titleText]="'Beyond business'"
+[paragraphTexts]="['Some of that youthful energy is channeled beyond business. Through projects such as our coding school in Kakuma refugee camp, training teachers in pastoral Samburu and our open-source coding initiatives providing entry-level opportunities to hundreds of junior engineers', 'These projects don\'t necessarily have an immediate financial return, but their impact can be tangibly measured in the long-term.']"
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages/components/beyond-business/beyond-business.component.html:4]: Challenge. Set the values as properties in your component and bind to that instead.

Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages/components/beyond-business/beyond-business.component.html:4]: Challenge. Set the values as properties in your component and bind to that instead.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by LemmyMwaura** [comment]
The elewa image banner is not imported and also set the values being passed to it as properties

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**